### PR TITLE
Various bugfixes and styling improvements

### PR DESCRIPTION
**Pull request by tddumont**
This PR includes various bugfixes and styling improvements, including for the Missions page, the Goals page, the task overview, the Missions Progress page, the Task review page, and the nav drawer.

### CODE REVIEW

File: src/Components/MissionProgress/MissionProgress.js [hunk 1]
```diff
@@ -49,7 +49,10 @@ export default function MissionProgress() {
         for(var i = 0; i < nTasks; i++){
             if (progress[i].submission != null) completedCount++;
         }
-        return 100 * completedCount/nTasks;
+        if (nTasks > 0)
+            return 100 * completedCount/nTasks;
+        else 
+            return 0;
```
Comment (ConnorBoulais) [inline src/Components/MissionProgress/MissionProgress.js:55]: nice job fixing this bug

**Review by ConnorBoulais — APPROVED** [review]
Looks good, great finalizing changes before beta

File: src/App.js [hunk 2]
```diff
@@ -168,11 +168,11 @@ function App() {
   return (
     <div>      
       <div className="navbar">
-        <p className="title">flipt.ED</p>
+        <p className="title" onClick={() => (hist.push(HOME_SCREEN_PATH))}>flipt.ED</p>
```
Comment (ryanmadamba) [inline src/App.js:171]: Nice addition

File: src/Components/TaskReview/TaskReview.js [hunk 3]
```diff
@@ -14,65 +14,46 @@ export default function TaskReview(props) {
...
-        <div className='options-bar'>
-        <h2 onClick = {() => {onClickQuizReview()}} style = {oBarStyleArray[0]}>Quiz Review</h2>
-        <h2 onClick = {() => {onClickTaskResults()}} style = {oBarStyleArray[1]}>Task Results</h2>
-        <h2 onClick = {() => {onClickGetHelp()}} style = {oBarStyleArray[2]}>Get Help</h2>
+  function ReviewTabs() {
+    return (
+        <div className="tabButtonContainer" style={{width: "450px"}}>
+          <button className={activeTab==1 ? "tabButton" : "tabButtonNotActive" } onClick={()=>setActiveTab(1)}>Quiz Review</button>
+          <button className={activeTab==2 ? "tabButton" : "tabButtonNotActive" } onClick={()=>setActiveTab(2)}>Task Results</button>
+          <button className={activeTab==3 ? "tabButton" : "tabButtonNotActive" } onClick={()=>setActiveTab(3)}>Get Help</button>
```
Comment (ryanmadamba) [inline src/Components/TaskReview/TaskReview.js:24]: Nice

**Review by ryanmadamba — APPROVED** [review]
Looks great, Ted!

**Pull request merged**